third-party/reddit/arcane-artillery/gunslinger.xml

```diff
@@ -0,0 +1,1289 @@
+			<ul>
+				<li>
+					<i>(a)</i> scale mail or <i>(b)</i> leather, a flintlock pistol and 10 musketballs</li>
+				<li>
+					<i>(a)</i> a wheellock gun and 20 musketballs, <i>(b)</i> a blunderbuss and 20 spreadshot rounds, or <i>(c)</i> a pepperbox and 20 musketballs</li>
+				<li>
+					<i>(a)</i> two simple weapons, <i>(b)</i> a flintlock pistol and 10 musketballs, or <i>(c)</i> a shield</li>
+				<li>
+					<i>(a)</i>a dungeoneer's pack, <i>(b)</i> an explorer's pack, or <i>(c)</i> a scholar's pack</li>
+			</ul>
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_THE_TONGUE" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_QUICKDRAW" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_GUN_STUNTS" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_KNACK" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_FIGHTING_STYLE" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_SLINGERS_TRAIL" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_LUCKY_ITEM" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_ABILITYSCOREIMPROVEMENT" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_EXTRA_ATTACK" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_BULLET_TIME" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_EVASION" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_SHOOTOUT_SENSE" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_OVERWATCH" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_FINAL_STAND" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_SUPERHUMAN_REFLEXES" />
+			<div element="ID_RDDT_AA_CLASS_FEATURE_GUNSLINGER_GUNSLINGING_SUPREME" />
+		</description>
+		<sheet display="false">
+			<description>A master of firearms, showing their power with skill and style.</description>
+		</sheet>
+		<setters>
+			<set name="short">A master of firearms, showing their power with skill and style.</set>
+			<set name="hd">d8</set>
+		</setters>
+		<rules>
+			<grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_LIGHT_ARMOR" requirements="!ID_RDDT_AA_MULTICLASS_GUNSLINGER" />
+			<grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_MEDIUM_ARMOR" requirements="!ID_RDDT_AA_MULTICLASS_GUNSLINGER" />
+			<grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_SHIELDS" requirements="!ID_RDDT_AA_MULTICLASS_GUNSLINGER" />
+			<grant type="Proficiency" name="ID_RDDT_AA_PROFICIENCY_WEAPON_PROFICIENCY_LONGARMS" requirements="!ID_RDDT_AA_MULTICLASS_GUNSLINGER" />
+			<grant type="Proficiency" name="ID_RDDT_AA_PROFICIENCY_WEAPON_PROFICIENCY_SIDEARMS" requirements="!ID_RDDT_AA_MULTICLASS_GUNSLINGER" />
+			<grant type="Proficiency" name="ID_PROFICIENCY_WEAPON_PROFICIENCY_SIMPLE_WEAPONS" requirements="!ID_RDDT_AA_MULTICLASS_GUNSLINGER"  />
+			<grant type="Proficiency" name="ID_INTERNAL_PROFICIENCY_WEAPON_GROUP_CROSSBOW" requirements="!ID_RDDT_AA_MULTICLASS_GUNSLINGER"  />
```

Comment:
update `ID_INTERNAL_PROFICIENCY_WEAPON_GROUP_CROSSBOW` to `ID_INTERNAL_PROFICIENCY_GROUP_CROSSBOWS` for now to avoid the error in the new build.

Weapon group proficiencies aren't really a thing yet (most of the time the book lists a couple of weapons instead of a group) it was something i was testing for custom content. (and appending custom weapon proficiencies to these groups)